packages/theme-data/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@hig/theme-data-poc",
+  "version": "0.0.6-alpha",
+  "description": "HIG Theme Data",
+  "author": "Autodesk Inc.",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Autodesk/hig.git"
+  },
+  "module": "build/index.es.js",
+  "files": [
+    "build/*"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test": "react-scripts test --env=jsdom",
+    "build": "hig-scripts-build && babel-node scripts/buildThemes",
+    "lint": "hig-scripts-lint"
+  },
+  "devDependencies": {
+    "@hig/babel-preset": "^0.1.0",
+    "@hig/eslint-config": "^0.1.0",
+    "@hig/scripts": "^0.1.0",
+    "@hig/typography": "^0.1.1",
+    "babel-cli": "^6.26.0",
```

Comment:
_Future:_ we should make the build script more flexible to handle builds like these.